views/interface.js

```diff
@@ -0,0 +1,65 @@
+function getPostData() {
+    fetch("https://chitter-backend-api-v2.herokuapp.com/peeps").then(response => {
+    response.json().then(post => {
+    post.forEach((element, index, array) => {
```

Comment:
does the array argument here let you access the entire array as a single object? that's super cool and really useful (I could have used that in mine!)